addons/web/static/src/legacy/scss/bootstrap_overridden.scss

```diff
@@ -60,6 +60,9 @@ $variable-prefix: '' !default;
 // Restore negative margins disabled in BS5 by default
 $enable-negative-margins: true !default;
 
+// Enable CSS Grid classes
+$enable-cssgrid: true !default;
```

Comment:
At first sight I don't see any advantage of using `grid` for this layout.
Should we reconsider it?